README.md

```diff
@@ -1,6 +1,3 @@
-# Project Title
-Lorem ipsum dolor sit amet.
-
```

Comment:
Why remove this? It's intended to be a "template" for the downstream project. I think it's good practive to have a title and short description in the README. We could add an installation step "change title and description in `README.md`"...

Reply:
Ah, I somehow thought that snuck in and wasn't supposed to be there. Makes a lot of sense, added it back in and slightly rephrased it!

Reply:
> the lorem ipsum wasn't too clear

Ah, sorry: I thought it was universally known as blind text, i.e. please replace